xknx/secure/keyring.py

```diff
@@ -233,6 +233,7 @@ class Keyring(AttributeReader):
 
     def __init__(self) -> None:
         """Initialize the Keyring."""
+        self.backbone = None
```

Comment:
```suggestion
        self.backbone: XMLBackbone | None = None
```
The class level type annotation above could be removed then.